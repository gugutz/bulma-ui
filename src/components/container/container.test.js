import React from 'react'
import renderer from 'react-test-renderer'
import Container from './container'

describe('<Container /> component', () => {
  it('should render correctly', () => {
    const tree = renderer
      .create(
        <Container>
          <div>some content here</div>
        </Container>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isFluid prop', () => {
    const tree = renderer
      .create(
        <Container isFluid>
          <div>some content here</div>
        </Container>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isWidescreen prop', () => {
    const tree = renderer
      .create(
        <Container isWidescreen>
          <div>some content here</div>
        </Container>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })

  it('should support isFullhd prop', () => {
    const tree = renderer
      .create(
        <Container isFullhd>
          <div>some content here</div>
        </Container>
      )
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
