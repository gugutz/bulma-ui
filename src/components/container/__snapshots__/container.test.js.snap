// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Container /> component should render correctly 1`] = `
<div
  className="container"
>
  <div>
    some content here
  </div>
</div>
`;

exports[`<Container /> component should support isFluid prop 1`] = `
<div
  className="container is-fluid"
>
  <div>
    some content here
  </div>
</div>
`;

exports[`<Container /> component should support isFullhd prop 1`] = `
<div
  className="container is-fullhd"
>
  <div>
    some content here
  </div>
</div>
`;

exports[`<Container /> component should support isWidescreen prop 1`] = `
<div
  className="container is-widescreen"
>
  <div>
    some content here
  </div>
</div>
`;
